cmd/agent/daemon/daemon.go

```diff
@@ -38,14 +39,17 @@ func NewRunCommand(version string) *cobra.Command {
 		logRateInterval = pflag.Duration("log-rate-iterval", 100*time.Millisecond, "Log rate limit interval")
 		logRateBurst    = pflag.Int("log-rate-burst", 100, "Log rate burst")
 
-		sendLogLevel                 = pflag.String("send-logs-level", "", "send logs level")
-		containerdSockPath           = pflag.String("containerd-sock", "/run/containerd/containerd.sock", "Path to containerd socket file")
-		metricsHTTPListenPort        = pflag.Int("metrics-http-listen-port", 6060, "metrics http listen port")
-		pyroscopeAddr                = pflag.String("pyroscope-addr", "", "Enable pyroscope tracing")
-		hostCgroupsDir               = pflag.String("host-cgroups", "/cgroups", "Host /sys/fs/cgroups directory name mounted to container")
+		sendLogLevel          = pflag.String("send-logs-level", "", "send logs level")
+		containerdSockPath    = pflag.String("containerd-sock", "/run/containerd/containerd.sock", "Path to containerd socket file")
+		metricsHTTPListenPort = pflag.Int("metrics-http-listen-port", 6060, "metrics http listen port")
+		pyroscopeAddr         = pflag.String("pyroscope-addr", "", "Enable pyroscope tracing")
+		hostCgroupsDir        = pflag.String("host-cgroups", "/cgroups", "Host /sys/fs/cgroups directory name mounted to container")
+
+		containerStatsEnabled        = pflag.Bool("container-stats-enabled", false, "Enable container stats scraping")
 		containerStatsScrapeInterval = pflag.Duration("container-stats-scrape-interval", 60*time.Second, "Container resources scrape interval")
 
 		btfPath                        = pflag.String("btf-path", "/sys/kernel/btf/vmlinux", "btf file path")
+		ebpfEventsEnabled              = pflag.Bool("ebpf-events-enabled", false, "Enable ebpf events")
```

Comment:
I guess we should then enable eBPF events by default in the `values.yaml`. What do you think?

Reply:
The issue here that if only netflow collection is installed from cost network page we don't want to enable ebpf events tracing by default.

Reply:
Good point! We just need to remember that this is a breaking change though.